### PR TITLE
Feature/natural earth

### PR DESCRIPTION
**Pull request by manuelroth**
- Added natural earth data for upper zoomlevels (mostly 0 to 6)
- Improved scaleranks of country, marine labels, lakes
- Added layer state label back
- Replaced water polygons on level 0 to 5 with natural earth data

### CODE REVIEW

File: open-streets.tm2source/data.yml [hunk 1]
```diff
@@ -47,6 +93,7 @@ Layer:
     fields: 
       class: String
       osm_id: Number
+      type: String
```
Comment (lukasmartinelli) [inline open-streets.tm2source/data.yml]: Good we found out that type was missing. :+1:

File: open-streets.tm2source/data.yml [hunk 2]
```diff
@@ -629,20 +837,21 @@ Layer:
           coalesce(NULLIF(name_fr, ''), name) AS name_fr,
           coalesce(NULLIF(name_de, ''), name) AS name_de,
           coalesce(NULLIF(name_ru, ''), name) AS name_ru,
-          coalesce(NULLIF(name_zh, ''), name) AS name_zh
+          coalesce(NULLIF(name_zh, ''), name) AS name_zh,
+          scalerank
```
Comment (lukasmartinelli) [inline open-streets.tm2source/data.yml]: We don't need `scalerank` for water labels.

File: src/update-scaleranks/update.sql [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        ne.name ILIKE osm.name_en OR
+        ne.namealt ILIKE osm.name OR
+        ne.namealt ILIKE osm.name_en
+    )
+    AND (osm.type = 'city' OR osm.type = 'town' OR osm.type = 'village')
+    AND st_dwithin(ne.geom, osm.geometry, 50000)
+    ) AS improved_places
+WHERE osm_places.osm_id = improved_places.osm_id;
+
+UPDATE osm_water_polygons
```
Comment (lukasmartinelli) [inline src/update-scaleranks/update.sql]: No longer need the scalerank here.

**Comment by klokan** [comment]
Is there a preview of world for 0 to 6 or 7? Could you please upload one on Google Drive?

**Comment by manuelroth** [comment]
@klokan I uploaded a mbtiles of the world for z0 - z7. It looks the best with the newest OSM Bright style.

**Comment by klokan** [comment]
Thans @manuelroth. The zooming and ranking looks quite good - but there seems to be some issues on GL version generalisation of land shapes. See screenshot and the live version bellow:
screen shot 2015-11-15 at 21 38 16
http://osm.tileserver.com/open-streets/bright-v8world.html
compared to:
http://osm.tileserver.com/open-streets/bright-v8v6.html

Let's keep this in mind for GL optimisation #71, please.

**Comment by manuelroth** [comment]
@klokan Right now we are not doing any generalization on the land shapes. It seams Mapbox GL can't handle all of these details we provide. I will have a look into this.

**Pull request merged**